docs/CIP/CIP-XXXX/messages/client/subscribe.md

```diff
@@ -84,36 +72,26 @@ Sent by client to server right after establishing a connection. New clients must
                 }
               }
             },
-            "required": [
-              "credentials", "config"
-            ]
+            "required": ["credentials", "extensions"]
           }
         },
-        "required": [
-          "blockchain",
-          "publicKey",
-          "signature"
-        ]
+        "required": ["blockchain", "signature"]
       }
     },
     "timestamp": {
       "type": "string",
```

Comment:
I suggest we use `epochs` instead of `date-time`. It doesn't need timezone or any format. It will be much simpler to construct the message and implement the server-side verification.
```suggestion
      "type": "integer",
```